src/common/media.cpp

```diff
@@ -272,6 +275,13 @@ void media_loop(void) {
                     }
                     break;
                 }
+                while (len == MAX_CMD_SIZE && buffer[MAX_CMD_SIZE - 1] != '\n') {
+                    // Read until the end of the line
+                    if (!f_gets(buffer, sizeof(buffer), &media_print_fil)) {
```

Comment:
And what will be the state of `media_current_position` when the line is too long? Could you please add some unit tests so that we can verify these potential edge cases?